src/components/UnauthenticatedRoute/index.tsx

```diff
@@ -17,19 +19,27 @@ const querystring = (name, url = window.location.href) => {
   return decodeURIComponent(results[2].replace(/\+/g, " "));
 }
 
-const UnauthenticatedRoute = ({ component: C, props: cProps, ...rest }) => {
+const UnauthenticatedRoute = ({ component: C, ...rest }) => {
   const redirect = querystring("redirect");
   return (
     <Route
       {...rest}
       render={props =>
-        !cProps.isAuthenticated
-          ? <C {...props} {...cProps} />
+        !props.isAuthenticated
+          ? <C {...props} />
           : <Redirect
               to={redirect === "" || redirect === null ? "/" : redirect}
             />}
     />
   );
 };
 
-export default UnauthenticatedRoute;
+const mapStateToProps = state => ({
+  isAuthenticated: state.isAuthenticated
+})
+
+const mapDispatchToProps = dispatch => ({
+  userHasAuthenticated: isAuthenticated => dispatch(authenticateUser(isAuthenticated))
```

Comment:
I don't see this method `userHasAuthenticated` is used anywhere. Can you please check?

Reply:
@leo-le-07